### ISSUE TITLE
Safari showing error "Can't find variable: Intl" after rendering.

### ISSUE THREAD

**Issue by amitgaikwad712** [angular/universal#516]
Only on the safari  after server data is rendered it is showing "ReferenceError: Can't find variable: Intl" Error. then all the data and images are not getting appeared on the screen...

**Comment by jimmythesaint82** [comment]
You need some polyfills.

import 'intl';
and what locals you need.
import 'intl/locale-data/jsonp/en';

or

<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=Intl.~locale.en"></script>

**Comment by jimmythesaint82** [comment]
This error also occurs on mobile

**Comment by amitgaikwad712** [comment]
Thanks jimmythesaint82 for Replay....
       This Problem has been solved.. The problem was in the code. I am trying to apply date pipe on the "20 May 2016" string that could be result in the that "Intl" error.

**Comment by MarkPieszak** [comment]
Is this error happening only during server side rendering, or is it an Angular Core issue itself?

**Comment by amitgaikwad712** [comment]
it an Angular Core issue itself.

**Comment by MarkPieszak** [comment]
Ahh ok! If there isn't an issue about it, open one there so they know about it :)
Glad you got it working.

**Comment by linusbrolin** [comment]
Sorry to post in a closed issue, but I thought it better than openaing a new one.
I'm having a similar issue with Intl on the server, but instead of getting an error message, I cant get the formatters to use any locale other than english on the server.
Formatting on the client is fine, so it's just a server issue.

If I install the Intl package and load the polyfills according to the guide:
https://www.npmjs.com/package/intl

**Comment by tinchou** [comment]
I'm seeing this error on:

Safari on iOS 9
Firefox 49 on Android Nougat

Not on:
Edge 38 on Windows 10
Safari on iOS 10
Any Chrome browser I tested, mobile or desktop

I came to this repo for a solution because I'm using `angular2-universal-polyfills`, and so I ditched `core-js` polyfills. Should I be adding this polyfill myself? Should this be polyfilled out of the box on your package?

I haven't isolated what is failing, but I can dig deeper if you want to fix it. For now I'll try to add `core-js` back and find which polyfill fixes this.
